content/en/logs/guide/aws-eks-fargate-logs-with-kinesis-data-firehose.md

```diff
@@ -0,0 +1,178 @@
+---
+title: AWS EKS Fargate logs with Kinesis Data Firehose
+kind: guide
+further_reading:
+- link: https://docs.datadoghq.com/logs/log_configuration/processors/
+  tag: Documentation
+  text: Processors 
+- link: https://docs.aws.amazon.com/eks/latest/userguide/fargate-logging.html
+  tag: Documentation
+  text: Fargate logging
+- link: https://docs.aws.amazon.com/eks/latest/userguide/fargate-profile.html
+  tag: Documentation
+  text: AWS Fargate profile
+---
+
+## Overview
+
+AWS Fargate on EKS provides a fully managed experience for running Kubernetes workloads. Kinesis Data Firehose can be used with EKS's Fluentbit log router to collect logs in Datadog.
+
+{{< img src="logs/guide/aws-eks-fargate-logs-with-kinesis-data-firehose/log_streaming_diagram.png" alt="Diagram of the log flow depicting a Fargate EKS cluster sending container logs through Fluentbit log router to Kinesis data firehose and an S3 backup bucket within AWS and then on to Datadog" responsive="true">}}
+ 
+   
+### Kinesis Data Firehose vs Cloudwatch log forwarding {#firehose-vs-cloudwatch}
+
+The following are key differences between using Kinesis Data Firehose and CloudWatch log forwarding.
+
+- **Metadata and tagging**: Metadata such as Kubernetes namespace and container ID are accessible as structured attributes when sending logs with Kinesis Data Firehose.
+
+- **AWS Costs**: AWS Costs may vary for individual use cases but Kinesis Data Firehose ingestion is generally less expensive than comparable Cloudwatch Log ingestion. 
+
+## Requirements
+1. An EKS cluster with a [Fargate Profile][1]. In this guide, the cluster is named `fargate-cluster` with a fargate profile nameed `fargate-profile` applied to the namespace `fargate-namespace`.
+2. The following command line tools: [`kubectl`][6], [`aws`][7], [`jq`][8]
```

Comment:
Just following the instructions myself, I have `kubectl` and the AWS CLI installed, but wasn't able to download `jq`, as this URL doesn't seem to be a trusted source. Is there another URL that should be used for obtaining `jq`, or alternatively, another way to do this?

Reply:
I figured out a way to remove the `jq` usage here :-)